sdk/digitaltwins/Azure.ResourceManager.DigitalTwins/src/Azure.ResourceManager.DigitalTwins.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-preview.1</Version>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- NuGet props -->
+    <PackageId>Azure.ResourceManager.DigitalTwins</PackageId>
+    <Description>Azure management client SDK for Azure resource provider Microsoft.DigitalTwins</Description>
+    <PackageTags>azure;management;digitaltwins</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Is this project reference needed?

Reply:
After rebase on master, it wouldn't build, but fixed once I added it.

Reply:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'HttpPipeline' could not be found (are you missing a using directive or an assembly reference?)	Azure.ResourceManager.DigitalTwins (netstandard2.0)	C:\code\Azure\asfn1\sdk\core\Azure.Core\src\Shared\AutoRest\ArmOperationHelpers.cs	30	N/A
Error	CS0246	The type or namespace name 'RequestMethod' could not be found (are you missing a using directive or an assembly reference?)	Azure.ResourceManager.DigitalTwins (netstandard2.0)	C:\code\Azure\asfn1\sdk\core\Azure.Core\src\Shared\AutoRest\ArmOperationHelpers.cs	33	N/A
Error	CS0246	The type or namespace name 'Response' could not be found (are you missing a using directive or an assembly reference?)	Azure.ResourceManager.DigitalTwins (netstandard2.0)	C:\code\Azure\asfn1\sdk\core\Azure.Core\src\Shared\AutoRest\ArmOperationHelpers.cs	42	N/A
```